src/Microsoft.VisualStudio.Editors/MyExtensibility/MyExtensibilityPropPage.vb

```diff
@@ -222,24 +231,25 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 #Region "Windows Form Designer generated code"
         Friend WithEvents labelDescription As Label
         Friend WithEvents linkLabelHelp As VSThemedLinkLabel
-        Friend WithEvents listViewExtensions As MyExtensionListView
-        Friend WithEvents colHeaderExtensionName As ColumnHeader
+        Friend WithEvents DataGridViewExtensions As MyExtensionDataGridView
```

Comment:
fields should normally start with a lower case, but it's unfortunate that all of these are `Friend` (`internal` in C#).  @davkean do you know why that is?

Reply:
Default for VB.